apps/store/src/pages/new-sitemap-pending.xml.js

```diff
@@ -35,12 +36,17 @@ const getFilteredPages = async () => {
   let perPage = 100
   let paginating = false
 
+  const cacheVersion = STORYBLOK_CACHE_VERSION ? parseInt(STORYBLOK_CACHE_VERSION) : NaN
```

Comment:
Maybe we can do this?

```suggestion
  const cacheVersion = parseInt(STORYBLOK_CACHE_VERSION)
```